src/articles/2024-01-21_DDEV-for-js.md

```diff
@@ -0,0 +1,327 @@
+---
+title: Node.js Development with DDEV
+created: 2024-01-21T00:00:00-05:00
+type: article
+permalink: "articles/ddev-for-js/"
+tags: [drupal,ddev,node.js]
+summary: ""
+---
+
+For much of the Drupal community, I imagine, [DDEV](https://ddev.com/) is a familiar tool for local development. It allows developers that want to work with their own sandboxed version of a website to spin up a set of containers that behave like a regular server-hosted copy of the site. The local version gets a nice URL, has its own database, and can run PHP all in the container - regardless of whether the host machine has that software installed. While DDEV is pretty ubiquitious in CMS communities like Drupal, WordPress, and Typo3, it's perhaps not as familiar to developers working in Node.js, Python, or other decoupled setups. This blog post is for _those_ developers.
+
+## What is DDEV?
+
+Straight from [the documentation](https://ddev.readthedocs.io/en/stable/), "DDEV is an open source tool for launching local web development environments in minutes. It supports PHP, Node.js, and Python (experimental)." Furthermore, "DDEV is a Go application that stores its configuration in files on your workstation. It uses those blueprints to mount your project files into Docker containers that facilitate the operation of a local development environment. DDEV writes and uses docker-compose files for you, which is a detail you can cheerfully ignore unless you’re Docker-curious or defining your own services."
+
+As a front-end developer with no interest or experience in DevOps, the key feature for _why_ you should consider DDEV is standardized environments for every member of a team working on the same project.
```

Comment:
```suggestion
As a front-end developer with no interest or experience in DevOps, the key feature for _why_ you should consider DDEV is standardized environments for every member of a team working on the same project.
```

Sure, but I think this just makes the argument for using VMs or containerized solutions in general. Why not docker compose, or lando, or whatever?

Perhaps something like "As a front-end developer, I've found DDEV to be a great solution because it lets us have standardized environments for every team member, yet those team members don't have to have interest or experience in DevOps work".